settings.py

```diff
@@ -21,6 +21,6 @@
 
 USE_I18N = True
 
-USE_TZ = False
+USE_TZ = True
```

Comment:
Ensure that `USE_TZ` is set to `True` only if you want Django to use timezone-aware datetimes. This is generally recommended, but make sure it aligns with your project's requirements.